OpenRA.Game/Map/MapCache.cs

```diff
@@ -128,7 +128,10 @@ public void LoadMaps()
 			foreach (var kv in MapLocations)
 			{
 				foreach (var map in kv.Key.Contents)
+				{
 					LoadMapInternal(map, kv.Key, kv.Value, mapGrid, null, modDataRules);
+					GC.Collect();
```

Comment:
I guess its a compromise between CPU or memory overhead.
But to be honest I put that as a first attempt to reduce memory usage, it might not be necessary anymore.
I will try without and if it work I will just drop it. I dont think it make sense to put it outside the loop because the memory big-bang happened within that loop.

Reply:
removed